dashboard/settings.py

```diff
@@ -191,6 +191,7 @@
         'PASSWORD': ENV.get('MYSQL_PASSWORD', 'student_dashboard_password'), # password for user
         'HOST': ENV.get('MYSQL_HOST', 'localhost'),
         'PORT': ENV.get('MYSQL_PORT', 3306),
+        'OPTIONS': {'charset': 'utf8mb4'},
```

Comment:
I was using `django-mysql` package ( for Mysql DB caching support for LTI )locally the warning keeps on poping up it is not encoded with `utf8mb4_unicode_ci`. Once I set this option the warning went away

Reply:
Great! Yeah that package has a check for this, but it doesn't have a fix yet for it. Good to know that this fixes that.